src/test/java/com/marklogic/kafka/connect/source/ConstraintStorageTest.java

```diff
@@ -0,0 +1,87 @@
+package com.marklogic.kafka.connect.source;
+
+import com.marklogic.client.document.XMLDocumentManager;
+import com.marklogic.client.io.StringHandle;
+import com.marklogic.junit5.PermissionsTester;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+
+class ConstraintStorageTest extends AbstractIntegrationSourceTest {
```

Comment:
To be consistent with story/action-based test names, let's do "StoreConstraintValueInMarkLogicTest".